.github/workflows/release_asan_clang.yml

```diff
@@ -41,6 +41,19 @@ jobs:
 
     runs-on: ubuntu-20.04-self-hosted
 
+    container:
```

Comment:
Why did you do this? It is already run in the container.

Reply:
fixed